apps/admin/backend/src/tabulation/write_ins.test.ts

```diff
@@ -596,6 +596,15 @@ test('filterOvervoteWriteInsFromElectionResults', async () => {
       votes: { 'zoo-council-mammal': ['lion'] },
       card: { type: 'bmd' },
     },
+    {
+      ballotStyleGroupId: '1M' as BallotStyleGroupId,
+      batchId: 'batch-1-1',
+      scannerId: 'scanner-1',
+      precinctId: 'precinct-1',
+      votingMethod: 'precinct',
+      votes: { 'zoo-council-mammal': ['write-in-unmarked', 'lion'] },
+      card: { type: 'bmd' },
+    },
```

Comment:
This test adds an Unmarked Write-in overvote. It should increase the count of Lion by 1 before adjudication, and should have no effect on counts after adjudication + filtering overvotes. 

That is why line 657 does increment by 1, as Lion has 1 more vote before adjudication, but 771 does not increment, as Lion vote count remains the same after adjudication.

And that is why checks on 668 and 766 do not change, as the unmarked write-in does not contribute to tallies before adjudication nor after adjudication/overvote filtering.